app/components/RelatedRecordTable.tsx

```diff
@@ -0,0 +1,56 @@
+import Card from "react-bootstrap/Card";
+import Table from "react-bootstrap/Table";
+import RelatedRecordTableRow from "./RelatedRecordTableRow";
+import { ColDataCardDef } from "@/types/types";
+
+interface RelatedRecordTableProps<T extends Record<string, unknown>> {
```

Comment:
since i mentioned prop type declarations in an earlier comment, did you decide to use an interface here because of the `Record` type or as a stylistic choice since there are more props in this component? Just curious to learn more.